tutorials/intro-to-geospatial-analysis/Sample.js

```diff
@@ -0,0 +1,132 @@
+//  Geometries
+Map.setCenter(-88, 41.8, 9);
+function multi(feature) {
+ //  Reduce number of vertices in geometry; the number is to specify maximum error in meters
+ var simple = feature.simplify(1000);
+ //  Find centroid of geometry
+ var center = simple.centroid();
+ //  Create buffer around geometry; the number represents the width of buffer in meters
+ var buff = center.buffer(500);
+ //  return variable from function
+ return buff;
+}
+//  Load Feature Collection of the dissolved boundary of Chicago stored in assets
+var Chic = ee.FeatureCollection('users/tirthankar25/chicago_diss');
+//  Load Feature Collection of the neighborhood-scale Chicago data stored in assets
+//  var Chic_coll=ee.FeatureCollection('users/tirthankar25/chicago');
+//  Map function over Feature Collection
+//  var Res=Chic_coll.map(multi);
+//  Add the layer to the map
+// Map.addLayer(Res);
+Map.addLayer(Chic, {}, 'Chicago dissolved');
+//  Find the rectangle that emcompasses the southernmost, westernmost, easternmost, and northernmost
+//  points of the feature
+//  var bound = Chic.geometry().bounds();
+//  Find the polygon covering the extremities of the feature
+//  var convex = Chic.geometry().convexHull();
+//  Map.addLayer(bound, {},'Chicago bounds');
+//  Map.addLayer(convex, {},'Chicago convex hull');
+//  Find the area common to two or more features
+//  var Intersect=bound.intersection(convex,100);
+//  Map.addLayer(Intersect, {},'Bound and convex intersection');
+//  Find the area encompassing two or more features; number is the maximum error in meters
+//  var Union=bound.union(convex,100);
+//  Map.addLayer(Union, {},'Bound and convex union');
+//  Find area of feature
+//  var Ar=Chic.geometry().area();
+//  Find length of line geometry
+//  var length=Chic.geometry().length();
+//  Find permeter of feature
+//  var peri=Chic.geometry().perimeter();
+//  print (peri)
```

Comment:
Did you mean for this block of code to be commented out?

Reply:
The way it worked in the original workshop module I had was that I would block out parts of code to demonstrate different functions, since it is, as far as I know, not possible to run just blocks of code using the Javascript API (correct me if I am wrong). If the commenting out is an issue, I can just have one script that includes, in sequence, all permutations of operations I used in the workshop.

Reply:
I removed the comments now.